vendor/golang.org/x/sys/execabs/execabs_go119.go

```diff
@@ -15,3 +15,7 @@ import (
 func isGo119ErrDot(err error) bool {
 	return errors.Is(err, exec.ErrDot)
 }
+
+func isGo119ErrFieldSet(cmd *exec.Cmd) bool {
+	return cmd.Err != nil
+}
```

Comment:
The code patch adds a new function `isGo119ErrFieldSet` which returns true if the `Err` field of the passed `*exec.Cmd` instance is not nil. 

I don't see any immediate bug risks with this code patch, but a couple of improvement suggestions are:
- Add comments to explain what the function does and when it should be used.
- Use more descriptive names for the function and its parameters, such as `hasExecCmdError(cmd *exec.Cmd) bool`.